scripts/flash/get-timestamp.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var encodeTag = require("../../src/format/tag/encode-tag");
+
+function getTime(augur, auth, callback) {
+  var timestamp = augur.api.Controller.getTimestamp();
+  var controller = augur.contracts.addresses[augur.rpc.getNetworkID()].Controller;
+
+  augur.api.Controller.lookup({ meta: auth, tx: {to: controller}, _key: encodeTag("Time")}, function (err, timeAddress) {
+    callback({
```

Comment:
Callbacks should use Node.js style: first argument is an error (or `null` if there is no error), second argument contains the results.